appinventor/components-ios/src/BarChartDataModel.swift

```diff
@@ -62,8 +77,10 @@ open class BarChartDataModel: Chart2DDataModel {
       let flooredX = Int(floor(x))
       return BarChartDataEntry(x: Double(flooredX), y: y)
     } else {
-      // Handle number format exception
-      return nil
+      let dateX = formatDateTime(rawX) ?? 0.0
+      let y = Double(rawY) ?? 0.0
```

Comment:
Is casting to 0.0 on failure how the Android code works? I though we raised an error via the form.